persistence/mapper/src/main/scala/net/liftweb/mapper/MappedField.scala

```diff
@@ -363,20 +356,18 @@ trait MappedField[FieldType <: Any,OwnerType <: Mapper[OwnerType]] extends Typed
    */
   def readPermission_? = false
 
-
-
   /**
    * Assignment from the underlying type.  It's ugly, but:<br />
    * field() = new_value <br />
    * field set new_value <br />
    * field.set(new_value) <br />
    * are all the same
    */
-  def update[Q <% FieldType](v: Q) {
+  def update[Q](v: Q)(implicit implFn: Q => FieldType): Unit = {
     this.set(v)
   }
 
-  def apply[Q <% FieldType](v: Q): OwnerType = {
+  def apply[Q](v: Q)(implicit implFn: Q => FieldType): OwnerType = {
```

Comment:
Why is the implicit required in this new version?

Reply:
`Q <% FieldType` is a view bound syntax which is deprecated in Scala 2.13.x

> * **View bounds deprecated**
>   * Deprecated: A <% B (#6500)
>   * Use instead: (implicit ev: A => B)

https://github.com/scala/scala/releases/tag/v2.13.0

Reply:
Ah, good to know